eclair-core/src/test/scala/fr/acinq/eclair/integration/MessageIntegrationSpec.scala

```diff
@@ -77,6 +84,7 @@ class MessageIntegrationSpec extends IntegrationSpec {
     val probe = TestProbe()
     val eventListener = TestProbe()
     nodes("B").system.eventStream.subscribe(eventListener.ref, classOf[OnionMessages.ReceiveMessage])
+    sleep(100 millis)
```

Comment:
There should be a check that probe did receive a SendOnionMessageResponse afer `alice.sendOnionMessage` just below ?